fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckMessageCard.kt

```diff
@@ -0,0 +1,215 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.shopping.ui
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.LinkText
+import org.mozilla.fenix.compose.LinkTextState
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.compose.button.PrimaryButton
+import org.mozilla.fenix.theme.FirefoxTheme
+
+private val cardShape = RoundedCornerShape(8.dp)
+
+/**
+ * Review Quality Check message UI.
+ *
+ * @param title The title of the message.
+ * @param messageType The [ReviewQualityCheckMessageType] of message to display.
+ * @param modifier Modifier to be applied to the card.
+ * @param description The optional secondary piece of text.
+ * @param linkText An optional piece of text with a clickable substring.
+ * @param hyperlinkText The text within [linkText] that is clickable.
+ * @param linkURL The optional URL to return when the link is clicked.
+ * @param buttonText The text to show in the optional button.
+ * @param onLinkClick Invoked when the link is clicked. When not-null, the link will be visible.
+ * @param onButtonClick Invoked when the button is clicked. When not-null, the button will be visible.
+ * @param icon The UI to display before the text, typically an image or loading spinner.
+ */
+@Composable
+fun ReviewQualityCheckMessageCard(
```

Comment:
```suggestion
fun ReviewQualityCheckInfoCard(
```
This is to align with component naming in the figma. Quick reference: [figma](https://www.figma.com/file/VE5xayz3yOSB5qOATUU9Im/Mobile-Shopping-Experience-MVP?node-id=4698%3A74610&mode=dev)